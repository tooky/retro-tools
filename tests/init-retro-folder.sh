retro init
ls doc/retro
cat doc/retro/README.md
