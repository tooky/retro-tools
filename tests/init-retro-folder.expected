retro init
doc/retro/README.md
ls doc/retro
README.md
cat doc/retro/README.md
# Daily Retros

We are running daily retrospectives in an effort to "turn up the good" â€“ Woody Zuill.

Each retro we run is recorded in this folder as a dated Markdown file.

To find the retrospective for 23rd May 2020 you would look in the file `2020/05/23.md`
